Messaging.md

```diff
@@ -364,11 +392,13 @@ Note that `outcome_i` is the outcome value itself and not its hash that will be
 This type contains information provided by an oracle on an event that it will attest to.
 See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 
-#### Version 0 `oracle_event`
+**For backward compatibility reasons, this type is currently serialized as a TLV. It is expected to be changed to the new serialization format in a near future.**
+
+#### `oracle_event`
 
-1. type: 55330 (`oracle_event_v0`)
-2. data:
-   * [`u16`:`nb_nonces`]
+1. type: 55330
+1. data:
+   * [`bigsize`:`nb_nonces`]
```

Comment:
this could theoretically break compatability i believe. I'm not sure if it matters in practice though.

Reply:
Reverted to u16 (I think it does matter)